source/partials/_speakers.html.erb

```diff
@@ -0,0 +1,12 @@
+<section class="speakers">
+  <ul class="speaker-list">
+    <% data.speakers.people.each do |s| %>
```

Comment:
`s`? Rename it to `speaker` 😤

Reply:
hahahauuha but rubyistas don't like this?

Reply:
short name is bad for any programming language :P